SU2_CFD/include/solvers/CNEMOEulerSolver.hpp

```diff
@@ -282,72 +297,67 @@ class CNEMOEulerSolver : public CFVMFlowSolverBase<CNEMOEulerVariable, COMPRESSI
+   * \brief Update the solution using an explicit Euler scheme.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
+   */
+  void ExplicitEuler_Iteration(CGeometry *geometry, CSolver **solver_container, CConfig *config) final;
 
   /*!
-     * \brief Update the solution using an explicit Euler scheme.
-     * \param[in] geometry - Geometrical definition of the problem.
-     * \param[in] solver_container - Container vector with all the solutions.
-     * \param[in] config - Definition of the particular problem.
+   * \brief Update the solution using a general explicit RK scheme.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
    * \param[in] iRKStep - Runge-Kutta step.
-     */
+   */
   void ExplicitRK_Iteration(CGeometry *geometry, CSolver **solver_container,
-                            CConfig *config, unsigned short iRKStep) override;
+                            CConfig *config, unsigned short iRKStep) final;
 
   /*!
-     * \brief Update the solution using an implicit Euler scheme.
-     * \param[in] geometry - Geometrical definition of the problem.
-     * \param[in] solver_container - Container vector with all the solutions.
-     * \param[in] config - Definition of the particular problem.
-     */
-  void ImplicitEuler_Iteration(CGeometry *geometry, CSolver **solver_container, CConfig *config) override;
-
-  /*!
-     * \brief Set the total residual adding the term that comes from the Dual Time Strategy.
-     * \param[in] geometry - Geometrical definition of the problem.
-     * \param[in] solver_container - Container vector with all the solutions.
-     * \param[in] config - Definition of the particular problem.
-     * \param[in] iRKStep - Current step of the Runge-Kutta iteration.
-     * \param[in] iMesh - Index of the mesh in multigrid computations.
-     * \param[in] RunTime_EqSystem - System of equations which is going to be solved.
-     */
-  void SetResidual_DualTime(CGeometry *geometry, CSolver **solver_container, CConfig *config,
-                            unsigned short iRKStep, unsigned short iMesh, unsigned short RunTime_EqSystem) override;
+   * \brief Update the solution using the classical RK4 scheme.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] iRKStep - Runge-Kutta step.
+   */
+  void ClassicalRK4_Iteration(CGeometry *geometry, CSolver **solver_container,
+                              CConfig *config, unsigned short iRKStep) final;
```

Comment:
This was missing from NEMO and the incompressible solver, any special reason?

Reply:
Thats interesting, there shouldn't be a reason to it is missing.